bc_obps/registration/schema/v1/regulated_products.py

```diff
@@ -10,3 +10,4 @@ class RegulatedProductSchema(ModelSchema):
     class Meta:
         model = RegulatedProduct
         fields = ["id", "name", "unit", "is_regulated"]
+        from_attributes = True
```

Comment:
I don't think this is necessary, since we're already listing the fields.
On a related note, the v1 API is now deprecated, so making changes to its code should really be limited to fixing security issues